doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -6,131 +6,410 @@ Cite Test Guide
+
+   #. ``Enable tests with multiple namespaces`` tests included
+
+      .. image:: ./image/tewfs-1_0.png
 
 Run WFS 1.1 tests
 -----------------
 
 .. important::
 
-   Running WFS 1.1 tests require PostGIS to be installed on the system.
+   Running WFS 1.1 tests require PostgreSQL with PostGIS extension installed to be installed on the system.
 
 Requirements:
 ~~~~~~~~~~~~~
+- GeoServer
+- teamengine
 - Posgresql
 - PostGIS
 
+#. Prepare the environment:
+
+   - login in postgresql and create a user named "cite".
+
+   .. code:: sql
+
+     createuser cite;
 
-#. Create a PostGIS user named "cite":
+   - Create a database named "cite", owned by the "cite" user:
 
-   .. code:: SQL
+   .. code:: sql
 
-     createuser cite
+     createdb cite own by cite;
 
-#. Create a PostGIS databased named "cite", owned by the "cite" user:
+   - enter to the database and enable the postgis extension:
 
-   .. code:: SQL
+   .. code:: sql
 
-     createdb -T template_postgis -U cite cite
+    create extension postgis;
 
-#. Change directory to the ``citewfs-1.1`` data directory and execute the script ``dataset-sf0.sql``:
+   - Change directory to the citewfs-1.1 data directory and execute the script dataset-sf0-postgis2.sql:
 
-   .. code:: SQL
+   .. code-block:: shell
 
-     psql -U cite cite < dataset-sf0.sql
+    cd <path of geoserver repository>
+    psql -U cite cite < build/cite/wfs11/citewfs-1.1/dataset-sf0-postgis2.sql
 
-#. Start GeoServer with the ``citewfs-1.1`` data directory. Example:
+   - Start GeoServer with the citewfs-1.1 data directory. Example:
 
-   .. code:: SHELL
+   .. important::
 
-     cd <root of geoserver install>
-     export GEOSERVER_DATA_DIR=<root of geoserver sources>/build/cite/wfs11/citewfs-1.1
-     ./bin/startup.sh
+     If the postgresql server is not in the same host of the geoserver, you have to change the `<entry key="host">localhost</entry>` in the `datastore.xml` file, located inside each workspace directory. ex.
 
-#. Change directory back to the cite tools and run the tests:
+     .. note::
 
-   .. code:: SHELL
+       <path of geoserver repository>/data/citewfs-1.1/workspaces/cgf/cgf/datastore.xml
 
-     ant wfs-1.1
+   .. code-block:: shell
+
+    cd <path of geoserver install>
+    export GEOSERVER_DATA_DIR=<path of geoserver repository>/build/cite/wfs11/citewfs-1.1
+    ./bin/startup.sh
+
+
+#. Start the test:
+
+   .. code:: shell
 
-   With the following parameters:
+     make webUI
 
-   #. ``Capabilities URL`` http://localhost:8080/geoserver/wfs?service=wfs&request=getcapabilities&version=1.1.0
+#. Go to the browser and open the teamengine `webUI <http://localhost:8888/teamengine>`_.
+
+   - click on the **Sign in** button and enter the user and password.
+
+   - after created the session, and chose the test. Enter the following parameters:
+
+   #. ``Capabilities URL`` http://geoserver:8080/geoserver/wfs?service=wfs&request=getcapabilities&version=1.1.0
 
    #. ``Supported Conformance Classes``:
```

Comment:
Actually GeoServer supports also locking, so it should be:

![image](https://user-images.githubusercontent.com/325433/114538712-f580a700-9c53-11eb-9e61-d4e6a8287ef8.png)

Reply:
solved in https://github.com/randomorder/geoserver/pull/3/commits/91da4cbdc686ba5397014ab7887381ee240f8a1b